x-pack/plugins/cases/server/client/cases/utils.ts

```diff
@@ -76,6 +77,13 @@ const getCommentContent = (comment: CommentResponse): string => {
   } else if (comment.type === CommentType.alert || comment.type === CommentType.generatedAlert) {
     const ids = getAlertIds(comment);
     return `Alert with ids ${ids.join(', ')} added to case`;
+  } else if (comment.type === CommentType.actions) {
+    const firstHostname = comment.actions.targets[0]?.hostname;
+    const totalHosts = comment.actions.targets.length;
+    const actionText = comment.actions.type === 'isolate' ? 'Isolated' : 'Released';
```

Comment:
~do we need to to use I18n translations?~ Let's create an issue here to convert this string with i18n translation
@cnasikas @jonathan-buttner what do you think?

Reply:
Yeah there's a number of strings that are surfaced to the user from the backend that aren't translated. I'll create a ticket.